silverstripe/framework/CVE-2019-19325.yaml

```diff
@@ -0,0 +1,11 @@
+title:     'CVE-2019-19325: XSS through non-scalar FormField attributes'
+link:      https://www.silverstripe.org/download/security-releases/cve-2019-19325/
+cve:       CVE-2019-19325
+branches:
+    4.0.x:
+        time:     2020-02-16 23:54:40Z
+        versions: ['>=4.0.0', '<4.4.5']
+    4.5.x:
+        time:     2020-02-16 23:54:40Z
+        versions: ['>=4.5.0', '<4.5.2']
```

Comment:
silverstripe/installer 4.5.1 ships with framework 4.5.2. That's not a mistake.

Robbie tagged framework 4.4.1 out of sync 2 weeks ago.